.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Description
+<!-- Describe your changes in detail -->
+
+## Motivation and context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link the issue here. -->
+
+## How has this been tested?
+<!-- Please describe how you tested your changes. -->
+
+## Screenshots (if appropriate)
+
+## Types of changes
+<!-- What types of changes does your code introduce? -->
+<!-- Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing
+functionality to not work as expected)
+
+## Checklist
+<!-- See the following points, and put an `x` in all the boxes that apply. -->
+<!-- If you're unsure about any of these, please ask. We're here to help! -->
+- [ ] PR title provides summary of the changes and starts with one of the
+[pre-defined prefixes](../../utils/release.yml)
+<!-- For example: "doc: Add example to db.describe documentation" -->
+<!-- or if it is a fix use "db.describe: fix JSON output format" -->
+- [ ] My code follows the [code style](../../doc/development/style_guide.md)
+of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
```

Comment:
What's the difference between these two? They seem pretty much the same for a new external contributor.

Reply:
The first one intended: ".... but I need support to update the documentation." ?

Reply:
> The first one intended: ".... but I need support to update the documentation." ?

Ok, what if it were the second checkbox (of these two) and it said something similar to "I need help for other changes of the documentation" (or better wording), to imply that they still tried, but it is not enough to complete it.

Reply:
The alternative is something like "I have updated documentation (when applicable)" but when unchecked that doesn't tell us if the change is needed and was not done or it's not needed. Do you have better idea how to formulate "My change either doesn't require change in doc or it requires it and I did it."?